radcampaign/pulitzer2016
# Footer base

To test:

- Pull, revert, clear Drupal cache 
- Footer styles and responsive behavior should be in place.
